pytamaro/point_names.py

```diff
@@ -0,0 +1,63 @@
+"""
+Names of notable points (point of interest) that can be used as pinning positions of a graphic
+"""
+
+from pytamaro.point import point, translate, i_hat, j_hat
+
+
+center = point(0.0, 0.0)
```

Comment:
Maybe this should refer to a `zero` (the origin) part of the `point` module (just like the basis).